enterprise/api_client/enterprise_catalog.py

```diff
@@ -143,37 +143,53 @@ def delete_enterprise_catalog(self, catalog_uuid):
             return {}
 
     @JwtLmsApiClient.refresh_token
-    def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
+    def get_content_metadata(self, enterprise_customer_catalogs, catalogs_last_modified=None):
```

Comment:
So, this is an API call used in a bunch of external repositories. You can't just change the Interface to not have an enterprise_customer UUID.  You need to put that field back to avoid a bunch of unintended consequences.

Reply:
What I would do is add your new catalogs_last_modified optional parameter, and if enterprise_catalogs is none, keep the existing behavior where that means all catalogs.

Reply:
I don't disagree and we need to be careful with changing APIs, but I want to make sure I'm not missing something: what external repositories? The only one using edx-enterprise's get_content_metadata call is enterprise_reporting AFAICT (and yes this change as written would break it). Everything else is using Catalog's `get_content_metadata` endpoint.

Reply:
method has been reverted to be backwards compatible, the minor issue is now there's basically two use cases